app/helpers/importable_exportable_helper.rb

```
@@ -0,0 +1,437 @@
+  end
+
+
+  # --------------------------------------------------------------
+  # MAIN IMPORT WORKFLOW
+  #
+  # Creates a temporary file with normalized headers,
+  # then iterates through rows, importing them one by one.
+  #
+  # Duplicate objects are collected and returned.
+  # --------------------------------------------------------------
+  def try_import_records(file, headers, use_header, defaults = {})
```

RuboCop (1.86.1)

[convention] Expected 1 empty line between method definitions; found 2.

(Layout/EmptyLineBetweenDefs)